presto-native-execution/src/test/java/com/facebook/presto/nativeworker/AbstractTestNativeGeneralQueries.java

```diff
@@ -190,6 +190,34 @@ public void testFiltersAndProjections()
         assertQuery("SELECT if(orderkey % 2 = 0, quantity_by_linenumber) FROM orders_ex");
     }
 
+    @Test
+    public void testAnalyzeStats()
+    {
+        assertUpdate("ANALYZE region", 5 /* Expected number of rows updated */);
```

Comment:
> /* Expected number of rows updated */

Please, remove this comment. Adding such comments to each assertUpdate call will make code unreadable.